components/grid.css

```diff
@@ -0,0 +1,37 @@
+.grid-container {
+  display: grid;
+  min-height: 10rem;
+  grid-gap: 2rem;
+  margin: 10px;
+  padding: 10px;
+}
+
+.grid-two {
+  grid-template-columns: 1fr 1fr;
+}
+.grid-three {
+  grid-template-columns: 1fr 1fr 1fr;
+}
+
+.bgGrey {
+  background-color: var(--overlay-bg-color);
+}
+
+.bgred {
+  background-color: var(--btn-primary-color);
+}
+.bggreen {
+  background-color: var(--border-two);
+}
+.bgblue {
+  background-color: var(--white-color);
+}
+
+@media only screen and (max-width: 625px) {
+  .grid-two {
+    grid-template-columns: 1fr;
+  }
+  .grid-three {
+    grid-template-columns: 1fr 1fr ;
+  }
+}
```

Comment:
Is this the correct way to make them responsive? I think no media query is needed for the grids here.